src/alpine/3.6/helix/amd64/Dockerfile

```diff
@@ -43,7 +43,8 @@ RUN apk update && \
 
 # Install Helix Dependencies
 
-RUN python -m pip install --upgrade pip==19.0.2 && \
+RUN sudo ln -sf /usr/bin/python3 /usr/bin/python && \
```

Comment:
I wouldn't expect `sudo` to be necessary here?

Reply:
probably not, mostly just muscle memory from how I normally type link commands :)